bigquery_etl/cli/backfill.py

```diff
@@ -0,0 +1,225 @@
+        sys.exit(1)
+
+    path = Path(sql_dir)
+
+    query_path = path / project_id / dataset_id / table_id
+
+    if not query_path.exists():
+        click.echo(f"{project_id}.{dataset_id}.{table_id}" + " does not exist")
+        sys.exit(1)
+
+    backfill = Backfill(
+        entry_date=TODAY,
+        start_date=start_date.date(),
+        end_date=end_date.date(),
+        excluded_dates=[e.date() for e in list(exclude)],
+        reason=DEFAULT_REASON,
+        watchers=[watcher],
+        status=BackfillStatus.DRAFTING,
+    )
+
+    backfills = []
+
+    backfill_file = query_path / BACKFILL_FILE
+
+    if backfill_file.exists():
+        backfills = Backfill.entries_from_file(backfill_file)
+        for entry in backfills:
+            validate_duplicate_entry_dates(backfill, entry)
+            if entry.status == BackfillStatus.DRAFTING:
+                validate_overlap_dates(backfill, entry)
+
+    backfills.append(backfill)
+
+    backfill_file.write_text(
+        "\n".join(backfill.to_yaml() for backfill in sorted(backfills, reverse=True))
+    )
+
+    click.echo(f"Created backfill entry in {backfill_file}")
+
+
+@backfill.command(
+    help="""Validate backfill.yaml file format and content.
+
+    Examples:
+
+    ./bqetl backfill validate moz-fx-data-shared-prod.telemetry_derived.clients_daily_v6
+
+    \b
+    # validate all backfill.yaml files if table is not specified
+    Use the `--project_id` option to change the project to be validated;
+    default is `moz-fx-data-shared-prod`.
+
+    Examples:
+
+    ./bqetl backfill validate
+    """
+)
+@click.argument("qualified_table_name", required=False)
+@sql_dir_option
+@project_id_option("moz-fx-data-shared-prod")
+@click.pass_context
+def validate(
+    ctx,
+    qualified_table_name,
+    sql_dir,
+    project_id,
+):
+    """Validate backfill.yaml files."""
+    backfill_files = []
+
+    if qualified_table_name:
+        try:
+            match = QUALIFIED_TABLE_NAME_RE.match(qualified_table_name)
+            project_id = match.group("project_id")
+            dataset_id = match.group("dataset_id")
+            table_id = match.group("table_id")
+        except AttributeError:
+            click.echo(
+                "Qualified table name must be named like:"
+                + " <project>.<dataset>.<table>"
+            )
+            sys.exit(1)
+
+        path = Path(sql_dir)
+        query_path = path / project_id / dataset_id / table_id
+
+        if not query_path.exists():
+            click.echo(f"{project_id}.{dataset_id}.{table_id}" + " does not exist")
+            sys.exit(1)
+
+        backfill_file = path / project_id / dataset_id / table_id / BACKFILL_FILE
+        backfill_files.append(backfill_file)
+
+    else:
+        backfill_files = paths_matching_name_pattern(
+            None, sql_dir, project_id, [BACKFILL_FILE]
+        )
+
+    for file in backfill_files:
+        try:
+            validate_file(file)
+        except yaml.YAMLError as e:
```

Comment:
```suggestion
        except (yaml.YAMLError, ValueError) as e:
```

Looks like both exceptions get handled in the same way. Could just have a single except block in this case:
https://stackoverflow.com/a/6470452